data/campaigns/Son_Of_The_Black_Eye/units/Old_Orcish_Shaman.cfg

```diff
@@ -1,100 +1,16 @@
 #textdomain wesnoth-sotbe
 [unit_type]
     id=Old Orcish Shaman
+    [base_unit]
+        id=Orcish Sorcerer
+    [/base_unit]
     name= _ "Old Orcish Shaman"
-    race=orc
-    image="units/elder-orcish-shaman.png"
-    hitpoints=22
-    movement_type=orcishfoot
-    [defense]
-        village=30
-    [/defense]
-    movement=5
     experience=50
     level=1
-    alignment=chaotic
-    advances_to=null
-    {AMLA_DEFAULT}
     cost=21
-    usage=mixed fighter
     description= _ "Orcish Shamans are the guardians of orcish magic. Respected among orcish tribes, they form the Orcish Council, which makes important decisions for the whole Orcish community and arbitrates the many conflicts that arise between tribes of this argumentative race. Although physically weak (for orcs), Orcish Shamans are good spell casters and can curse their enemies, draining their life. Old Orcish Shamans are shamans that have mastered magic, although their physical strength has declined with age."
-    die_sound={SOUND_LIST:ORC_DIE}
-    {DEFENSE_ANIM "units/elder-orcish-shaman-defend-2.png" "units/elder-orcish-shaman-defend-1.png" {SOUND_LIST:ORC_HIT} }
     [attack]
-        name=staff
-        #textdomain wesnoth-units
-        description= _"staff"
-        type=impact
-        range=melee
+        # Reduce the staff's physical damage to reflect age
```

Comment:
Instead of reducing the unit's stats this way, how about giving the "Aged" trait "musthave" status for this particular unit? Seeing as it's "Old" and all... (if that makes it *too* weak, its base stats can be raised to compensate; the idea behind doing agedness as a trait is just for flavorfulness)

Reply:
However, if we're going with aged but otherwise a vanilla Orcish Sorcerer, I'd rather apply it directly to the specific unit in the scenario.

I'm not quite sure yet if we want to remove these stubs though.

Reply:
Well, yes, I also applied the trait directly to the unit in my case, too, but I also buffed the base stats as well to make up for it